manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -17,13 +17,12 @@ images:
   cacheserver: gcr.io/ml-pipeline/google/pipelines/cacheserver:dummy
   cachedeployer: gcr.io/ml-pipeline/google/pipelines/cachedeployer:dummy
 
-serviceAccountCredential: ""
 gcpDefaultConfigName: "gcp-default-config"
 
 managedstorage:
   enabled: false
-  cloudsqlInstanceConnectionName: null
-  gcsBucketName: null
-  databaseNamePrefix: '{{ .Release.Name | replace "-" "_" | replace "." "_" }}'
```

Comment:
removed this default value, because if we keep the default value, it would break backward compatibility when upgrading from earlier versions with the databaseNamePrefix left as empty string.